pkg/plugin/aggregation/run.go

```diff
@@ -150,14 +152,22 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 	}()
 
 	// 4. Launch each plugin, to dispatch workers which submit the results back
+	certs := map[string]*tls.Certificate{}
```

Comment:
This is a weird diff, but the point was that the loop over plugins did 2 things:
 - create cert
 - run plugin

If we can't create certs I think that is a completely different type of error condition and I'm OK with maintaining the existing behavior where the server does a hard return there. I pulled it into its own loop to more clearly separate the two operations so that we wouldn't have some plugins already started, fail to generate the next cert, then return from that function (without completing the function to ingest results, update status, etc)